documentation/asciidoc/computers/raspberry-pi/boot-http.adoc

```diff
@@ -0,0 +1,113 @@
+== HTTP Boot
+
+The network install feature uses HTTP over ethernet to boot the device into the xref:getting-started.adoc#using-raspberry-pi-imager[Raspberry Pi Imager] application. You can change how this is configured and make use of HTTP Boot for your own purposes.
+
+=== Requirements
+
+To use HTTP boot you need to use the LATEST / STABLE bootloader configuration, and update to the bootloader dated on or after 10 March 2022. HTTP boot only works over ethernet, so you need an Ethernet cable connected to the network, e.g. via a socket on the back of your router.
+
+=== Configuration Properties
+
+==== NET_INSTALL_ENABLED
+
+By default net install is enabled on Pi4 and Pi400. On CM boards it is disabled by default. 
+
+When net install is enabled it detects a keyboard and displays the Net Install screen on boot. You can switch this behaviour on and off with this property.
```

Comment:
Should "it detects a keyboard" be "it attempts to detect a keyboard" ? i.e. does net-install get skipped if no keyboard is detected?